resources/2024-10-dapr-bindings.md

```diff
@@ -0,0 +1,326 @@
+# Dapr Binding Implementation
+
+* **Author**: Lucas Peirone (@SoTrx)
+
+## Overview
+
+This design document proposes the addition of support for Dapr Bindings within Radius. By integrating Dapr Bindings, resources will be able to interact with external systems in a decoupled and scalable manner, facilitating better communication between services and external resources.
+
+## Terms and definitions
+
+[Dapr](https://github.com/dapr/dapr): Distributed Application Runtime. It provides "building blocks" for writing microservices, which are high level functionalities. 
+
+[Dapr Bindings](https://docs.dapr.io/developing-applications/building-blocks/bindings/bindings-overview/): A Dapr building block that allows interaction with external resources in a loosely coupled way. Bindings are used to interact with external resources such as databases, queues, and storage systems.
+
+Bindings are further subdivided into two types: input and output bindings:
+- Input bindings are used to receive events from an external resource.
+- Output bindings are used to send events to an external resource.
+
+
+## Objectives
+
+> **Issue Reference:** radius-project/radius#7960
+
+### Goals
+
+- Enable users to create, update, and delete Dapr Bindings using Radius.
+
+### Non goals
+
+- (out-of-scope) Managing resources used by Dapr Bindings. This includes databases, queues, and storage systems. By definition, Dapr Bindings are intended to be used for external resources.
+- (out-of-scope) Default recipes for Dapr Bindings. Each binding will have different requirements, so it doesn't make sense to have a single default recipe.
+
+### User scenarios (optional)
```

Comment:
I think these focus on user scenarios for the Dapr Bindings themselves, which is helpful, but I'd recommend adding some scenarios on how a user would use Radius to bring a Dapr Binding into their application

Reply:
(Recap meeting)
The user story will be updated to reflect that